apps/judicial-system/web/src/components/Modals/Modal/Modal.tsx

```
@@ -138,6 +160,68 @@ const Modal: FC<PropsWithChildren<ModalProps>> = ({
+    },
+    closed: {
+      translateY: 50,
+      opacity: 0,
+      transition: { duration: 0.2 },
+    },
+  }
+
+  useEffect(() => {
+    const handleKeyDown = (e: { key: string }) => {
+      if (e.key === 'Escape') {
+        onClose && onClose()
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)